Obsidian.API/Plugins/PluginAttribute.cs

```diff
@@ -8,6 +8,15 @@ namespace Obsidian.API.Plugins
     [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
     public class PluginAttribute : Attribute
     {
+        public PluginAttribute(string name, string version, string authors, string description, string projectUrl)
```

Comment:
Since the constructor is for an attribute, it's unnecessary, either:
- Make a constructor for initializing properties (also make them get-only, this is the issue with the introduction of this constructor)
- Remove the constructor again

Reply:
Hi Craft,

I have kept the constructor for initializing the properties and made the properties get only. Hope that's what you were asking.

Reply:
Alright, then this change is qualified. 👍

Reply:
I'm not a fan of making all these properties required via a constructor. What do the others think? @Tides